examples/3rdparty/python/BUILD

```diff
@@ -11,7 +11,14 @@ remote_sources(
   args=dict(
     include_relpath='include',
     lib_relpath='',
-    native_lib_names=['tensorflow_framework'],
+    native_lib_names = {
+      'darwin': ['tensorflow_framework'],
```

Comment:
Is this example building any C++ code?

Reply:
Yes, see https://github.com/pantsbuild/pants/tree/master/examples/src/python/example/tensorflow_custom_op!

Reply:
ok, the OSX tests are failing as expected :)

Reply:
Oh no, our CI is actually really screwed up right now. Will restart.